addons/mail/static/src/widgets/discuss/discuss.js

```diff
@@ -185,6 +185,7 @@ const DiscussWidget = AbstractAction.extend({
         }
         // Mark All Read
         if (
+            this.discuss.threadViewer &&
```

Comment:
Is it really necessary? I mean, naturally if discuss has a thread, then there's a view on this thread, right?

Reply:
In this flux, maybe. In general no: if discuss is not open it can still have a thread, but not display it.
Just check `_computeHasThreadView`. There's also the mobile stuff that has been moved to it.